metricbeat/tests/system/test_template.py

```diff
@@ -0,0 +1,55 @@
+import os
+import metricbeat
+import json
+from nose.plugins.skip import SkipTest
+
+
+class Test(metricbeat.BaseTest):
+
+    def test_export_template(self):
+        """
+        Test export template works and contains all fields
+        """
+
+        if os.name == "nt":
+            raise SkipTest
```

Comment:
Why?

Reply:
I never fully investigated. There seemed to be a path issue in the windows tests.